docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc

```diff
@@ -0,0 +1,43 @@
+= How to use in-memory database to manage test data
+
+Assuming there is a project containing VIVIDUS-based tests
+(e.g. https://github.com/vividus-framework/vividus-starter[VIVIDUS Starter])
+and the goal is to manage test data in the in-memory database (e.g.
+https://h2database.com/html/features.html#feature_list[H2 database]).
+The database is deployed on tests start, accessible from tests and shutdown on
+tests end. The test data can be created from SQL files located in the project.
+
+. Add the required dependencies (DB plugin, H2 database) to the project `build.gradle`
++
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-db', version: '{current-version}')
+implementation(group: 'com.h2database', name: 'h2', version: '1.4.200')
```

Comment:
I think that by using specific version pf h2 we put updating responsibility on our shoulders

Reply:
Was done similar to this: https://docs.vividus.dev/vividus/latest/guides/execute-sql-against-csv.html
Specified version works as described in this guide. The others - we can not be sure.

Reply:
we'll keep explicit version in a sakeness of simplicity